src/benchmark/file_io_write_micro_benchmark.hpp

```diff
@@ -0,0 +1,69 @@
+#include "micro_benchmark_basic_fixture.hpp"
+
+#include <fcntl.h>
+#include <numeric>
+#include <sys/mman.h>
+#include <sys/stat.h>
+
+namespace hyrise {
+
+const auto MB = uint32_t{1'000'000};
+
+class FileIOWriteMicroBenchmarkFixture : public MicroBenchmarkBasicFixture {
+public:
+  void SetUp(::benchmark::State& state) override {
+    NUMBER_OF_BYTES = state.range(0) * MB;
+    NUMBER_OF_ELEMENTS = NUMBER_OF_BYTES / sizeof(uint32_t);
+    data_to_write = generate_random_indexes(NUMBER_OF_ELEMENTS);
```

Comment:
Pls rename this function

Reply:
`generate_random_data`

Reply:
👍 but would prefer `generate_random_numbers`